workspace/src/locales/manual/en.json

```diff
@@ -0,0 +1,612 @@
+{
```

Comment:
Why do we have still 2 different language file for one language? This is still as difficult as before.